lib/tlTimeline/RenderOptions.h

```diff
@@ -0,0 +1,24 @@
+// SPDX-License-Identifier: BSD-3-Clause
```

Comment:
This reminds me that I need to look into converting the copyright headers to what the ASWF recommends:
https://github.com/AcademySoftwareFoundation/tac/blob/main/process/contributing.md#copyright-notice-format